### PR TITLE
Create issue forms

### PR DESCRIPTION
**Pull request by Dusch4593**
Hi @yangc95 and @sonnynomnom! For the past few weeks, I've been working with [GitHub issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates) to create a set of new issue forms (primarily) for Docs maintainers like ourselves.

We seem to have passed the Hacktoberfest-surge since we now have less than 5 open PRs (excluding this one and #265 ). That's why I think the time is now right to submit this PR and implement issue forms going forward. 

I created three forms: 

- Entry Request (for suggesting up to 3 entries/terms at a time)
- Entry Edit (for suggesting non-code edits for up to 3 entries/terms at a time)
- Bug Report (for bringing focus to any bugs or errors in code blocks or codebyte examples)

When you have a chance, you should give them a try and share your thoughts! 😃

### CODE REVIEW

**Review by yangc95 — APPROVED** [review]
I really like this new layout! Looks good to me 💪 😸 👍

File: .github/ISSUE_TEMPLATE/bug_reports.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: Brief description of bug in Docs code/codebytes
+title: "[Bug/Error]: "
+labels: ["bug"]
+assignees: "name(s) here"
```
Comment (yangc95) [inline .github/ISSUE_TEMPLATE/bug_reports.yml:5]:
```suggestion
assignees: "Name(s) here"
```

**Review by yangc95 — COMMENTED** [review]
Just some capitalization I thought would be appropriate. Up to you though~

**Comment by Dusch4593** [comment]
@yangc95 Good call! I think I'm going to leave the labels uncapped since that's how they're already set on this repo. But I'll capitalize the other things you'd suggested! 😄

**Comment by hughlilly** [comment]
Marking as "required" any of the options under the "For maintainers" heading means that non-maintainers have to tick those boxes in order to create issues using these templates.

https://github.com/Codecademy/docs/blob/83fda684e9dfb7e84ff03a00afd396afb5aa7628/.github/ISSUE_TEMPLATE/bug_reports.yml#L50-L55

https://github.com/Codecademy/docs/blob/83fda684e9dfb7e84ff03a00afd396afb5aa7628/.github/ISSUE_TEMPLATE/entry_edit.yml#L35-L40

https://github.com/Codecademy/docs/blob/83fda684e9dfb7e84ff03a00afd396afb5aa7628/.github/ISSUE_TEMPLATE/entry_request.yml#L32-L37

**Comment by Dusch4593** [comment]
@hughlilly Good catch! Wasn't thinking of that when I originally made these. 😌

**Comment by hughlilly** [comment]
No worries. It took me ages to work out the syntax to show a range of lines in an issue comment, so I've taught myself something at least!

It's a shame there's no way to require them if the user is of a certain access level.

**Comment by Dusch4593** [comment]
> It's a shame there's no way to require them if the user is of a certain access level.

For sure! That would be a nice feature! I also wish there was some mechanism for `dropdown` where if an "Other" option is selected, a textbox appears for the user to expand further.

**Comment by hughlilly** [comment]
> I also wish there was some mechanism for `dropdown` where if an "Other" option is selected, a textbox appears for the user to expand further.

It seems [this is easy to achieve using just HTML & CSS](https://adrianroselli.com/2021/12/under-engineered-dependency-questions.html)!

**Pull request merged**